src/components/date-picker/DatePicker.tsx

```diff
@@ -480,7 +480,7 @@ class DatePicker extends Component<
             }
             value={inputValue || ''}
             onChange={this.handleInputChange}
-            onFocus={() => this.setState({ showPicker: true })}
+            onClick={() => this.setState({ showPicker: true })}
```

Comment:
this won't impact keyboard nav?

Reply:
it is actually to prevent to open it with the keyboard ! :D we want only with the click

otherwise with Enter key

Reply:
this change is not related to the fix of the PR, I just embed it here